pkg/targets/reconciler/tektontarget/reaper.go

```diff
@@ -40,27 +40,27 @@ func reaperThread(ctx context.Context, r *reconciler) {
 
 	p, err := cloudevents.NewHTTP()
 	if err != nil {
-		log.Fatal("failed to create new HTTP based cloudevent: ", zap.Error(err))
+		log.Fatalw("failed to create new HTTP based cloudevent: ", zap.Error(err))
```

Comment:
When using structured logs the error will have its own _field_, we should remove the `: ` at the message.


minor nits: 
- When logging we should capitalize sentences (when passing errors around we do not capitalize).
- `CloudEvent` instead of `cloudevent`
- I am not convinced of the usage of `Fatal`, this code lives in the reconciler and will tear down the whole process. That said, that error is very unexpected to happen and it was there before your PR.

This also applies to the other log fixes in this PR.

```suggestion
		log.Fatalw("Failed to create new HTTP based CloudEvent", zap.Error(err))
```

Reply:
Unresolving just in case you didn't noticed the second suggestion.
All log writtings looks like this:

```go
log.Warnw("some message: ", zap.Error(err))
```

The colon should not be there, this is not `log.Warnf("failed to do something: %v", err)` but `log.Warnf` which will render an structured log, using the first string as the `msg` element, and the other parameters as structured details for the log, so `zap.Error(err)` will render as the `error` element. When using JSON it would render something like this:


```json
{"level":"warning","ts":"2021-12-14T13:35:42.152Z","msg":"failed to do something","error":"detailed reason why something failed",...}
```

As you can see, there is no need to add a colon to the `msg`